clubs.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Clubs
+priority: 4
+description: The UUU clubs directory
+image: assets/images/uniclub.jpg
+nav-menu: true
+menu_bottom: "test"
+show_tile: true
+---
+
+{% comment %} List of clubs {% endcomment %}
+{% assign sorted_clubs = site.data.clubs | sort:"name" %}
+<section class="spotlights">
+
+{% for club in sorted_clubs %}
+
+{% capture title %}
+{% if club.website %}
+## [{{club.name}}]({{club.website}})
+{% else %}
+## {{club.name}}
+{% endif %}
+{% endcapture %}
+
+{% capture titleHTML %}
+{{ title | markdownify }}
+{% endcapture %}
+
+{% capture filltext %}
+**Address:** [{{club.address}}](https://maps.google.co.uk/?q={{club.latitude}},{{club.longitude}})
```

Comment:
You should guard this with and if so we only show `Address:` if we have something to show.